config/services.yaml

```diff
@@ -268,6 +268,9 @@ services:
     App\Service\CasManager:
       public: true
 
+    App\Service\CsvWriter:
```

Comment:
I'm not sure why this is needed, something I'm not seeing? Or just leftover from development?

Reply:
you're right, no explicit wiring needed. i was just following suit. anyway, removed.